docs/vscode_monaco_upgrade.md

```diff
@@ -2,22 +2,22 @@
 
 ## Preparation
 
-- It is assumed `monaco-vscode-api`, `vscode` and `monaco-editor` repos are cloned locally on the same directory level. `monaco-editor` is not required, but helpful
+- It is assumed `monaco-vscode-api` and `vscode` and `monaco-editor`.
```

Comment:
The sentence is not complete. 🙂

Reply:
Wtf did I do? 😨 will update

Reply:
Fixed! also added 2 new commits to synchronize library versions